grails-app/domain/org/pih/warehouse/order/OrderItem.groovy

```diff
@@ -45,6 +45,9 @@ class OrderItem implements Serializable {
     Date dateCreated
     Date lastUpdated
 
+    Date currentDeliveryDate
```

Comment:
Please change these to match the names on Shipment i.e. estimatedDeliveryDate and actualDeliveryDate.